apps/aurora-portal/src/server/Compute/routers/keypairRouter.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod"
+import { protectedProcedure } from "../../trpc"
+import { keypairsResponseSchema, Keypair } from "../types/keypair"
```

Comment:
is key pair probably a reserver keyword?

Reply:
No, it's not!